products/compute/terraform.yaml

```diff
@@ -803,6 +803,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           {{description}}
 
           These are in the same namespace as the managed SSL certificates.
+      managed.domains: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: |-
```

Comment:
Do you mind breaking this out to a file? This is just a minor convention thing, we try to avoid inlined funcs generally.

Reply:
Done.